### PR TITLE
Links and books

### PR DESCRIPTION
**Pull request by topepo**
Fixed links to developer sections.

Also added more information on the books. @dgrtwo, @rudeboybert, @ismayc, and @juliasilge please suggest edits to the book summaries if you so choose.

### CODE REVIEW

File: content/books/moderndive/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: 'Statistical Inference via Data Science: A ModernDive into R and the tidyverse'
```
Comment (ismayc) [inline content/books/moderndive/index.md]:
```suggestion
title: 'Statistical Inference via Data Science: A ModernDive into R and the Tidyverse'
```

Comment (ismayc) [reply content/books/moderndive/index.md]: Publisher's decision here to capitalize

File: content/books/moderndive/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+authors: [Chester Ismay, Albert Y. Kim]
+---
+
+![](cover.png) 
+
+This book is intended to be a gentle introduction to the practice of analyzing data and answering questions using data the way data scientists, statisticians, data journalists, and other researchers would. Over the course of this book, you will develop your "data science toolbox," equipping yourself with tools such as data visualization, data formatting, data wrangling, and data modeling using regression.
```
Comment (ismayc) [inline content/books/moderndive/index.md]:
```suggestion
This book is intended to be a gentle introduction to the practice of analyzing data and answering questions using data the way data scientists, statisticians, data journalists, and other researchers would. Over the course of this book, you will develop your "data science toolbox," equipping yourself with tools such as data visualization, data formatting, data wrangling, data modeling using regression, and statistical inference via hypothesis testing and confidence intervals.
```

File: content/books/moderndive/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+An **HTML version** of this text can be found at [`https://moderndive.com/`](https://moderndive.com/). 
+
+The data sets and R code are available in the GitHub repository [`https://github.com/moderndive/ModernDive_book`](https://github.com/moderndive/ModernDive_book).
```
Comment (ismayc) [inline content/books/moderndive/index.md]:
```suggestion
The data sets and R code are available in the GitHub repository [`https://github.com/moderndive/ModernDive_book`](https://github.com/moderndive/ModernDive_book) and also the accompanying `moderndive` R package on [CRAN](https://cran.r-project.org/package=moderndive).
```

**Review by ismayc — COMMENTED** [review]
Just a few tweaks I've added as suggestions on GitHub here.

**Comment by juliasilge** [comment]
Edited the description and after that looks good to me.

Ready to merge from my perspective. 👍

**Pull request merged**